code/Test_definitions/quality-on-demand-createSession.feature

```diff
@@ -0,0 +1,374 @@
+Feature: CAMARA Quality On Demand API, v0.10.0-rc.1 - Operation createSession
```

Comment:
Are the changes between `v0.10.0-rc.1` and `v0.11.0-rc.2` already considered?

Reply:
Yes, it's a typo. Versions were aligned to r1.1. And they should be adapted to r1.2 as part of the PR, as with the API specs